deps/openssl/config/README.md

```diff
@@ -1,4 +1,4 @@
-## Upgrading OpenSSL-1.1.0
+## Upgrading OpenSSL-1.1.0 or OpenSSL-1.1.1
```

Comment:
Are we ever likely to upgrade to another 1.1.0 release now we have 1.1.1?

Reply:
No, though the process applies on 10.x.  It will only be done one more time on 10.x, though, to update to 1.1.1, so I'll just remove the 1.1.0 ref.